airflow/contrib/sensors/triggered_dagrun_sensor.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains the TriggeredDagSensor which allows for checking the status
+of triggered DAG runs."""
+from airflow.exceptions import AirflowException
+from airflow.utils import db
+from airflow.utils.state import State
+from airflow.utils.decorators import apply_defaults
+from airflow.utils.trigger_rule import TriggerRule
+from airflow.models import DagRun
+from airflow.sensors.base_sensor_operator import BaseSensorOperator
+
+
+class TriggeredDagRunSensor(BaseSensorOperator):
```

Comment:
```suggestion
class DagRunSensor(BaseSensorOperator):
```

Triggered is redundant - all dagruns are triggered by something - and this operator works with scheduled runs fine.

Reply:
Originally it was called DagRunSensor but @XD-DENG thought it would be clearer to emphasize that this sensor is meant to handle dagruns triggered via the DagRun operator.  I'm fine with changing the name back but wanted to make that clear.

Reply:
If you’re talking about this, https://github.com/apache/airflow/pull/4291#issuecomment-466609722